docs/en/observability/deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,164 @@
+
+. Add {aws} integration in {kib}.
+. Deploy function from {aws} SAR (Serverless Application Repository).
+
+[[aws-serverless-forwarder-deploy-kibana]]
+== Add {aws} integration in {kib}
+
+. Go to **Integrations** in {kib} and search for {aws} (or select the **{aws}** category to filter the list).
+. Click the {aws} integration, select **Settings** and click **Install {aws} assets** to install all the {aws} integration assets.
+
+Adding integrations from {kib} provides appropriate pre-built dashboards, ingest node configurations, and other assets that help you get the most out of the data you ingest. For more information, refer to the https://docs.elastic.co/en/integrations[Integrations documentation].
+
+// add screenshot?
+
+[[aws-serverless-forwarder-deploy-sar]]
+== Deploy elastic-serverless-forwarder from SAR
+
+There are several deployment methods available via {aws} SAR:
+
+* <<aws-serverless-forwarder-deploy-console>>
+* <<aws-serverless-forwarder-deploy-cloudformation>>
+* <<aws-serverless-forwarder-deploy-terraform>>
+
+[[aws-serverless-forwarder-deploy-console]]
+=== Deploy using {aws} Console
+
+. Login to {aws} console and navigate to the Lambda service.
+. Click **Create function**.
+. Select **Browse serverless app repository**.
+. Select **Public applications**.
+. Search for **elastic-serverless-forwarder** and select it from results.
++
+[role="screenshot"]
+image::images/aws-serverless-forwarder-create-function.png[Create Elastic Serverless Forwarder Lambda function within SAR]
++
+. Complete the **Application settings** as follows:
+    * `ElasticServerlessForwarderS3ConfigFile` set the value of the S3 URL in the format `s3://bucket-name/config-file-name` to the configuration file for your deployment of Elastic Serverless Forwarder - this will populate the `S3_CONFIG_FILE` environment variable of the deployed Lambda.
+    * `ElasticServerlessForwarderSSMSecrets` add a comma delimited list of {aws} SSM Secrets ARNs referenced in the config yaml file (if any).
+    * `ElasticServerlessForwarderKMSKeys` add a comma delimited list of {aws} KMS Keys ARNs to be used for decrypting {aws} SSM Secrets referenced in the config yaml file (if any).
+    * `ElasticServerlessForwarderSQSEvents` add a comma delimited list of Direct SQS queues ARNs to set as event triggers for the Lambda (if any).
+    * `ElasticServerlessForwarderS3SQSEvents` add a comma delimited list of S3 SQS Event Notifications ARNs to set as event triggers for the Lambda (if any).
+    * `ElasticServerlessForwarderKinesisEvents` add a comma delimited list of Kinesis Data Stream ARNs to set as event triggers for the Lambda (if any).
+    * `ElasticServerlessForwarderCloudWatchLogsEvents` add a comma delimited list of Cloudwatch Logs Log Groups ARNs to set subscription filters on the Lambda (if any).
+    * `ElasticServerlessForwarderS3Buckets` add a comma delimited list of S3 buckets ARNs that are sources for the S3 SQS Event Notifications (if any).
```

Comment:
> > config yaml
> 
> I recommend consistently using either "`config.yaml` file" or "config YAML file".

Yeah, good call - sticking with `config.yaml` file

> 
> > Cloudwatch Logs Log Groups ARNs
> 
> This might be right, but it looks strange. 🙃

It did and it does - had another look at AWS docs and I think 'Cloudwatch Logs log group ARNs' is used and slightly better...